rust/web_prover/src/notarize.rs

```diff
@@ -126,9 +126,7 @@ pub async fn notarize(params: NotarizeParams) -> Result<(Attestation, Secrets, R
         );
     }
 
-    let prover = prover_task.await??;
-
-    let mut prover = prover.start_notarize();
+    let mut prover = prover_task.await??;
```

Comment:
This implementation, along with the deprecated notarize method, follows the TLSNotary example:
https://github.com/tlsnotary/tlsn/blob/v0.1.0-alpha.11/crates/examples/attestation/prove.rs#L201-L217